rocky/reports/views/base.py

```diff
@@ -314,7 +314,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ReportTypeSelectionView(BaseReportView, ReportBreadcrumbs, TemplateView):
```

Comment:
TemplateView is removed, it clashes with ListView from the BaseOOILIst. use it in specific views where needed.